src/Http/ResponseSender.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Tempest\Http;
+
+interface ResponseSender
```

Comment:
This is the place where you'd add `#[InnitializedBy(ResponseSenderInitializer::class)]`

Reply:
I will revisit this. I had initially had it set up that way but it was not loading in. Not sure if I was doing something wrong or if something wasn't working right with discovery.

Reply:
This appears to be working. 👍🏼